src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
```

Comment:
You should avoid using static methods or static variables unless they are constants. In this case, RADIUS is not a constant that would be used across instances of FigureSupplier but seems to be a default value for a Circle radius. This should probably be a constant inside the Circle class or a parameter passed to the method that creates the Circle.